### DISCUSSION TITLE
StageWebView and IE11 end of life

### DISCUSSION THREAD

**Discussion by yan-campos** [airsdk/Adobe-Runtime-Support#1018] [Q&A]
Does anyone know or have information on what will happen with StageWebView and IE11 end of life in 2022? Will it continue to required IE11 when using the system native browser to true?

Trying to get information for clients, to be correct that Windows applications may or may not work after that date. We are working on evolving, but trying to have the correct communication. 

Thanks.

**Comment by ajwfrost** [comment]
StageWebView on Windows uses the MSHTML 'WebBrowser' control, and this will remain as part of Windows even after the IE11 browser application has been removed. This is how Edge's "IE Mode" feature works..

Microsoft have said this will continue until the end of 2029 at least:
https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support

For the MSHTML confirmation:
https://techcommunity.microsoft.com/t5/windows-it-pro-blog/internet-explorer-11-desktop-app-retirement-faq/ba-p/2366549
> What is the MSHTML (Trident) engine? How does that relate to IE mode? And what about WebOC? Or the MSHTA app? (Updated: June 29, 2021)
> The MSHTML (Trident) engine is the underlying platform for Internet Explorer 11. This is the same engine used by IE mode and it will continue to be supported (in other words, unaffected by this announcement). Both WebOC and the MSHTA app will continue to be supported as they rely on the MSHTML engine which is unaffected by this announcement. If you have a custom or third-party app that relies on the MSHTML platform, you can expect it to continue to work. For future app development, we recommend using WebView2.

**Reply by yan-campos** [reply]
Thank you!
